packages/fxa-auth-db-mysql/lib/db/schema/patch-110-111.sql

```diff
@@ -0,0 +1,205 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('110');
+
+-- Add a column to the accounts table to track the user's current
+-- ecosystemAnonId. For simplicity, we store this JWE as a base64 URL-safe
+-- string, and don't encode or decode it here.
+ALTER TABLE accounts
+  ADD COLUMN ecosystemAnonId TEXT CHARACTER SET ascii COLLATE ascii_bin DEFAULT NULL,
+ALGORITHM = INPLACE, LOCK = NONE;
+
+-- Update the account stored procedure to return `ecosystemAnonId` along
+-- with other `account` fields.
+CREATE PROCEDURE `account_8` (
+    IN `inUid` BINARY(16)
+)
+BEGIN
+    SELECT
+        a.uid,
+        a.email,
+        a.normalizedEmail,
+        a.emailVerified,
+        a.emailCode,
+        a.kA,
+        a.wrapWrapKb,
+        a.verifierVersion,
+        a.authSalt,
+        a.verifierSetAt,
+        a.createdAt,
+        a.locale,
+        a.lockedAt,
+        COALESCE(a.profileChangedAt, a.verifierSetAt, a.createdAt) AS profileChangedAt,
+        COALESCE(a.keysChangedAt, a.verifierSetAt, a.createdAt) AS keysChangedAt,
+        a.ecosystemAnonId
+    FROM
+        accounts a
+    WHERE
+        a.uid = inUid
+    ;
+END;
+
+-- Update the account record stored procedure to return `ecosystemAnonId` along
+-- with other `account` fields.
+CREATE PROCEDURE `accountRecord_7` (
+  IN `inEmail` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_bin
+)
+BEGIN
+    SELECT
+        a.uid,
+        a.email,
+        a.normalizedEmail,
+        a.emailVerified,
+        a.emailCode,
+        a.kA,
+        a.wrapWrapKb,
+        a.verifierVersion,
+        a.authSalt,
+        a.verifierSetAt,
+        a.createdAt,
+        a.locale,
+        a.lockedAt,
+        COALESCE(a.profileChangedAt, a.verifierSetAt, a.createdAt) AS profileChangedAt,
+        COALESCE(a.keysChangedAt, a.verifierSetAt, a.createdAt) AS keysChangedAt,
+        a.ecosystemAnonId,
+        e.normalizedEmail AS primaryEmail
+    FROM
+        accounts a,
+        emails e
+    WHERE
+        a.uid = (SELECT uid FROM emails WHERE normalizedEmail = LOWER(inEmail))
+    AND
+        a.uid = e.uid
+    AND
+        e.isPrimary = true;
+END;
+
+-- Update the email record stored procedure to return `ecosystemAnonId` along
+-- with other `account` fields.
+CREATE PROCEDURE `emailRecord_5` (
```

Comment:
FWIW, I hope to eventually depreciate the `emailRecord` procedure. It is limited because it only checks the account table for the email and verses `accountRecord` which pulls from the emails table. Long read but good description about reasons [here](https://github.com/mozilla/fxa/issues/512).